src/modules/commands/index.ts

```diff
@@ -52,6 +52,10 @@ export default class CommandsModule<T extends VoltareClient<any>> extends Voltar
    * @param command The command to register
    */
   register(command: any) {
+    if (Object.keys(command).length > 1) {
```

Comment:
Can you test for more than just the amount of keys? Probably type check if its an object as well and iterate the values to check if they are functions.

Reply:
I could try to validate if it is an object, but the other thing is already done by the same function, right?

Reply:
Well yeah, but might run into cases where exported variables are trying to be registered and then thrown as an error for that specific key.